### ISSUE TITLE
if table uuid  is same created by sql like 'create table xxx on cluster yyy'

### ISSUE THREAD

**Issue by linger000** [ClickHouse/ClickHouse#27161]
if table uuid  is same created by sql like 'create table xxx on cluster yyy'

**Comment by den-crane** [comment]
Can you rephrase your question?

```
CREATE TABLE test_atomic.rmt UUID '12345678-0000-4000-8000-000000000001' 
  ON CLUSTER cluster 
      (n UInt64, s String) 
ENGINE=ReplicatedMergeTree ORDER BY n
```

**Comment by linger000** [comment]
create table test1
 on cluster cluster1 (name String) 
ENGINE=MergeTree ORDER BY name

**Comment by den-crane** [comment]
UUID is the same for oncluster for Replicated ZK path, but not for Atomic table's UUID.
